rust/actyx/ax/src/cmd/users/devcert.rs

```diff
@@ -0,0 +1,48 @@
+use crate::{
+    certs::{AppDomain, DeveloperCertificateInput, ManifestDeveloperCertificate},
+    cmd::AxCliCommand,
+    crypto::{PrivateKey, PublicKey},
+    util::formats::{ActyxOSCode, ActyxOSResult, ActyxOSResultExt},
+};
+use futures::{stream::once, FutureExt, Stream};
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(version = crate::util::version::VERSION.as_str())]
+pub struct DevCertOpts {
+    /// The secret key used to sign the certificate
+    /// (this must match the AX_PUBLIC_KEY your `actyx` binary has been compiled with).
+    #[structopt(long, short = "A", env, hide_env_values = true)]
+    ax_secret_key: PrivateKey,
```

Comment:
The name changed from actyx_secret_key to ax_secret_key. Since it has `env`, this is a probable breaking change.